src/workerd/api/global-scope.h

```diff
@@ -382,6 +382,15 @@ struct ExportedHandler {
   jsg::Optional<jsg::Ref<ExecutionContext>> getCtx() {
     return ctx.map([&](jsg::Ref<ExecutionContext>& p) { return p.addRef(); });
   }
+
+  ExportedHandler clone(jsg::Lock& js) {
+    return ExportedHandler{
```

Comment:
You need to clone _all_ of the members, e.g. `fetch`, `trace`, etc.

Please make sure all of the tests pass with your compat flag forced to be enabled.

Reply:
How do I do that?